src/main/java/frc/robot/Robot.java

```diff
@@ -123,7 +123,7 @@ public void configureButtonBindings() {
         driveController.rightBumper().onTrue(drive.setSpeedCoef(0.25, 0.35)).onFalse(drive.setSpeedCoef(1, 1));
         driveController.leftBumper().onTrue(drive.setSpeedCoef(0.4000000001, 0.5)).onFalse(drive.setSpeedCoef(1, 1));
         driveController.back().onTrue(drive.resetGyroCommand());
-
+        driveController.a().whileTrue(drive.balance());
```

Comment:
We should probably remove this (or at least map it to a different button). We don't want Angel accidentally activating this.

Reply:
Fixed, it is now right trigger.